packages/tools/generator-widget/generators/app/templates/pluggable/native/fullTemplateTsFn/src/WidgetName.tsx.ejs

```diff
@@ -0,0 +1,23 @@
+import { ReactElement, createElement } from "react";
+
+import { Badge } from "./components/Badge";
+import { BadgeStyle } from "./ui/styles";
+import { <%- name %>Props } from "../typings/<%- name %>Props";
+
+export function <%- name %>(props: <%- name %>Props<BadgeStyle>): ReactElement | null {
+    const onClickHandler = (): void => {
```

Comment:
Isn't more logical to wrap this function into a useCallback as you are introducing functional components?